op_map.cc

```diff
@@ -3535,6 +3535,8 @@ static const std::map<int, createIOpMapItemFunc> reg = {
     REGISTER_OP_MAPPER(kTfLiteBuiltinSpaceToDepth, Space2DepthMapper),
     REGISTER_OP_MAPPER(kTfLiteBuiltinDepthToSpace, Depth2SpaceMapper),
     REGISTER_OP_MAPPER(kTfLiteBuiltinPrelu, PreluMapper),
+    REGISTER_OP_MAPPER(
+        kTfLiteBuiltinElu, SimpleOpMapper<tim::vx::ops::Gelu>, "Gelu"),
```

Comment:
```suggestion
        kTfLiteBuiltinGelu, SimpleOpMapper<tim::vx::ops::Gelu>, "Gelu"),
```

https://github.com/tensorflow/tensorflow/blob/0efbd0c271375da7c507cf290d226583f8199279/tensorflow/lite/builtin_ops.h#L180